index.html

```diff
@@ -8,10 +8,10 @@
     <title>FIRST CAPSTONE.</title>
 </head>
 <body class="full-body">
-    <img src="./images/Union.png" alt="hamburger-button" class="hamburger" onclick="openpopup()">
+    <img src="./images/Union.png" alt="hamburger-button" class="hamburger" id="openpopup">
     
     <div class="popup" id="popup">
-        <img src="./images/cloasepopup.svg" alt="close button" class="closepopup" onclick="closepopup()">
+        <img src="./images/cloasepopup.svg" alt="close button" class="closepopup" id="closepopup">
         <ul>
            <li><a href="./about.html">About</a></li> 
             <li><a href="./index.html">Home</a></li>
```

Comment:
- [ ] kindly make sure your navigation bar is in the right position and the design look like the design provides:grinning: :grinning: 
- [ ] You did a great job, your home page looks nice but in the desktop version there is no way to come back to your home page please add the link to your home page on your about page and make sure is working as expected 
below I provide clear information please make sure you implement it as required [ here ](https://github.com/elad237/capstone-project/pull/1/files#r1007155816) by the previous reviewer:grinning:  :grinning:  :grinning: 
- [ ] Kindly make sure your project contains a `Home` page and `About` page.
- [ ] It's possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. it means on your home page you will have the link to your about page and on the about page, you will have the link to your home page 
- [ ] Both pages contain the same navbar and footer.

Check the  [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to see the style and the container of your about page :+1: :grinning: :grinning: :grinning: